source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -2483,12 +2483,22 @@ void Graph::setDefaults(mx::InputPtr input)
     }
 }
 
-void Graph::addLink(ed::PinId inputPinId, ed::PinId outputPinId)
+void Graph::addLink(ed::PinId startPinId, ed::PinId endPinId)
```

Comment:
I found input vs. output, start vs. end a bit confusing. Have tried to make this more accurate by checking the input pin is actually an input pin.

Reply:
Thanks for making that change! It's a lot clearer now.